backend/backend/production_settings.py

```diff
@@ -173,6 +71,12 @@
     }
 }
 
+if TESTING:
```

Comment:
why throttling only under testing?
Anyway, I think throttling should be removed because now kubernetes will serve requests with the load balancer, making randomic the occurrence of throttle errors.

Reply:
I wouldn't remove it entirely, it's still some nice last line of defense if we're really getting hammered by a script.

Also, this isn't only setting it for testing, the lines above it still set it for the normal environment. This if statement _overwrites_ what's there only for the testing environment (so the tests don't throttle themselves).